gammapy/datasets/map.py

```diff
@@ -1572,7 +1572,7 @@ def info_dict(self, in_safe_data_range=True):
 
             if self.background:
                 summed_stat = self._counts_statistic[mask].sum()
```

Comment:
This already includes the mask, why the separate handling for the background?

Reply:
Because if we keep the change above: `npred_background` in `_count_statistic`, then the background field of info_dict become `npred_background`, but we already have a field `npred_background`. 
Does that answer your question @adonath ?

Reply:
I see it is for the backwards compatibility.